src/PrestaShopBundle/Resources/config/services/core/grid/doctrine_query_builder.yml

```diff
@@ -136,6 +136,7 @@ services:
             - "@=service('prestashop.adapter.legacy.context').getContext().shop.id"
             - '@prestashop.adapter.shop.context'
             - '@prestashop.adapter.feature.multistore'
+            - "@=service('prestashop.adapter.legacy.context').getContext().language.id"
```

Comment:
language.id ?

Reply:
Ooooh I totally messed up this PR 😨 

My initial code was good but it seems at the time to `git add` and `git commit` I mixed the PR with some other code changes

My bad. A total failure 😞